src/app/core/store/app-store.module.ts

```diff
@@ -76,9 +86,28 @@ const effects = [
   ViewsEffects
 ];
 
+function appStateReducer(reducer) {
```

Comment:
Cannot we move this to `app.reducer.ts` file?

Reply:
For some strange mysterious reason, if I move it to separate file, it doesn't work.